### DISCUSSION TITLE
Whiich folders need to be copied over to replicate Liferea settings?

### DISCUSSION THREAD

**Discussion by Ottaviocr** [lwindolf/liferea#1379] [General]
There's a .config/liferea/ , then a .cache/liferea/ . Do both folders need to be copied over from one computer to another? I copied only .config/liferea/  but the feeds were not replicated.

This is not a bug report, just a documentation question.

**Comment by lwindolf** [comment]
Sorry for replying so late. You need to copy only

- ~/.config/liferea
- ~/.local/share/liferea

the cache directory can be skipped.
